distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -908,6 +908,34 @@ type Column
     like self pattern =
         run_vectorized_binary_op self "like" (_ -> _ -> Error.throw (Illegal_State.Error "The `Like` operation should only be used on Text columns.")) pattern
 
+    ## Gets the year as a number from the date stored in the column.
+
+       Applies only to columns that hold the `Date` or `Date_Time` types.
+       Returns a column of `Integer` type.
+    year : Column -> Column
+    year self = Value_Type.expect_has_date self.value_type related_column=self.name <|
+        new_name = Naming_Helpers.function_name "year" [self]
+        run_vectorized_unary_op self "year" (_ -> Error.throw (Illegal_State.Error "Missing vectorized implementation for `year`. This is a bug in the Table library.")) new_name
+
+    ## Gets the month as a number (1-12) from the date stored in the column.
+
+       Applies only to columns that hold the `Date` or `Date_Time` types.
+       Returns a column of `Integer` type.
+    month : Column -> Column
+    month self = Value_Type.expect_has_date self.value_type related_column=self.name <|
+        new_name = Naming_Helpers.function_name "month" [self]
+        run_vectorized_unary_op self "month" (_ -> Error.throw (Illegal_State.Error "Missing vectorized implementation for `month`. This is a bug in the Table library.")) new_name
+
+    ## Gets the day of the month as a number (1-31) from the date stored in the
+       column.
+
+       Applies only to columns that hold the `Date` or `Date_Time` types.
+       Returns a column of `Integer` type.
+    day : Column -> Column
+    day self = Value_Type.expect_has_date self.value_type related_column=self.name <|
+        new_name = Naming_Helpers.function_name "day" [self]
+        run_vectorized_unary_op self "day" (_ -> Error.throw (Illegal_State.Error "Missing vectorized implementation for `day`. This is a bug in the Table library.")) new_name
```

Comment:
I guess inevitable but feels like these 3 could be extracted to a private function here.

Reply:
ok